packages/components/input/src/input.vue

```diff
@@ -443,6 +465,12 @@ export default defineComponent({
       emit('keydown', evt)
     }
 
+    const handlePaste = (evt: ClipboardEvent) => {
+      const preText = nativeInputValue.value
```

Comment:
BTW, can we add a dedicated test file against the `paste` event like others? Just to keep the code robust

Reply:
Added